plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -26,7 +26,7 @@ public static function load() {
 		add_action( 'pre_set_site_transient_update_themes', array( __CLASS__, 'transient_update_themes' ), 21, 1 );
 		add_action( 'upgrader_process_complete', array( __CLASS__, 'upgrader_process_complete' ) );
 		add_action( 'upgrader_pre_download', array( __CLASS__, 'block_expired_updates' ), 10, 2 );
-		add_action( 'plugins_loaded', array( __CLASS__, 'add_hook_for_modifying_update_notices' ) );
+		add_action( 'init', array( __CLASS__, 'add_hook_for_modifying_update_notices' ) );
```

Comment:
You're right, it's not. I was getting fatal errors from `WC_Woo_Update_Manager_Plugin::is_plugin_active()` though because `is_plugin_active_for_network` wasn't yet available. That can be addressed better.

Reply:
I changed this in 2fad07c2989be92be2b8c93e4d19cc00c28b876a, but not happy with it.

Actually, it would be even better to change this to `admin_init`, because the hooks there are only relevant for the admin, as they add admin notices.

Reply:
Changed in c5c3e262c86defcd212f725e5c44674ebfe25279